cloud-bigtable/README.MD

```diff
@@ -0,0 +1,23 @@
+# Google Cloud Bigtable via Apache HBase
+This initialization action installs Apache HBase libraries and the [Google Cloud Bigtable](https://cloud.google.com/bigtable/) [HBase Client](https://github.com/GoogleCloudPlatform/cloud-bigtable-client).
+
+
+## Using this initialization action
+You can use this initialization action to create a Dataproc cluster configured to connect to Cloud Bigtable:
+
+1. Create a Bigtable instance by following [these directions](https://cloud.google.com/bigtable/docs/creating-instance).
+1. Upload `bigtable.sh` to a Google Cloud Storage bucket `<GCS_BUCKET>`
```

Comment:
Do you want to give them some choices on how to do this?  Or point them at directions?